airflow/providers/databricks/hooks/databricks_base.py

```diff
@@ -175,21 +177,22 @@ def _parse_host(host: str) -> str:
 
     def _get_retry_object(self) -> Retrying:
         """
-        Instantiates a retry object
+        Instantiates a retry object.
         :return: instance of Retrying class
         """
         return Retrying(**self.retry_args)
 
     def _a_get_retry_object(self) -> AsyncRetrying:
         """
-        Instantiates an async retry object
+        Instantiates an async retry object.
         :return: instance of AsyncRetrying class
         """
         return AsyncRetrying(**self.retry_args)
 
     def _get_aad_token(self, resource: str) -> str:
         """
-        Function to get AAD token for given resource. Supports managed identity or service principal auth
+        Function to get AAD token for given resource.
+        Supports managed identity or service principal auth
```

Comment:
```suggestion
        Function to get AAD token for given resource.

        Supports managed identity or service principal auth.
```

I think there’s also a rule that the first line must not go over a certain length, and since reStructuredText has a line-collapsing rule, we need an empty line to prevent these being merged into a line.